Packs/DevSecOps/Integrations/DockerEngine/DockerEngine.yml

```
@@ -0,0 +1,1997 @@
+category: IT Services
+commonfields:
+  id: Docker Engine API
+  version: -1
+configuration:
+- display: Server URL (e.g. https://www.example.com:1000)
+  name: url
+  required: true
+  type: 0
+- display: Docker Client Certificate
+  hidden: false
+  name: client_certificate
+  required: true
+  type: 12
```

```suggestion
  type: 4
```
Please use encrypted type - 4